spec/controllers/users/address_autocomplete_controller_spec.rb

```diff
@@ -0,0 +1,86 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'support/shared/results_shared'
```

Comment:
include shared examples folder in `spec/rails_helper.rb`
also rename file to `shared_examples`